### DISCUSSION TITLE
Option "Last Updated"

### DISCUSSION THREAD

**Discussion by DjCodeGit** [vuepress-theme-hope/vuepress-theme-hope#1816] [Q&A] [answered]
Hello,

How does it work? For me, it doesn't work when i run my site in dev Mode.

On my pages in Guide, i never see the "last udapted" date.

Could you help me?

**Comment by Mister-Hope** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/33315834/167881888-28a6cff6-d063-4a0e-8042-7957e3f5b96c.png)

It would be nice if you can read the docs bro. https://vuepress-theme-hope.github.io/v2/guide/feature/meta.html#git-based-information

**Reply by DjCodeGit** [reply]
Thanks!!! i've not seen this chapters. sorry !!! 
But if we use the default Theme, it's work in dev Mode!!! :wink:
